airflow/config_templates/config.yml

```diff
@@ -598,7 +598,9 @@
       version_added: 2.0.0
       type: string
       example: ~
-      default: "{{{{ ti.dag_id }}}}/{{{{ ti.task_id }}}}/{{{{ ts }}}}/{{{{ try_number }}}}.log"
+      default: "dag_id={{{{ ti.dag_id }}}}/run_id={{{{ ti.run_id }}}}/task_id={{{{ ti.task_id }}}}/\
+               {{%% if ti.map_index != -1 %%}}map_index={{{{ ti.map_index }}}}/{{%% endif %%}}\
```

Comment:
It may be better to standardise the check to `map_index < 0` instead; IMO that’s more “logical” than specifically check for `-1`.

Of course, we shouldn’t have anything lower than -1, but in the very weird chance we hit one rouge value, it’s likely better to treat it as unmapped, since treating it as mapped would cause a crash later that’s difficult to debug.

Reply:
worth updating description key too so users are aware about map_index and https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/logging-tasks.html too

Reply:
`map_index >= 0` in this case, but yeah good idea.